tracker/tracker-assist/src/Mouse.ts

```diff
@@ -84,8 +87,63 @@ export default class Mouse {
     return null
   }
 
-  private readonly pScrEl = document.scrollingElement || document.documentElement // Is it always correct
-  private lastScrEl: Element | 'window' | null = null
+  startDrag(pos: XY) {
+    this.move(pos)
+    const el = document.elementFromPoint(pos[0], pos[1])
+    if (el) {
+      const downEvt = new MouseEvent("mousedown", {
+        bubbles: true,
+        cancelable: true,
+        clientX: pos[0],
+        clientY: pos[1],
+        buttons: 1,
+      });
+      el.dispatchEvent(downEvt);
+      this.isDragging = true;
+    }
+  }
+
+  drag(pos: XYDXDY) {
+    const [x, y, dx, dy] = pos
+    this.move([x, y]);
+
+    if (!this.isDragging) return;
+
+    const el = document.elementFromPoint(x, y);
+    if (el) {
+      const moveEvt = new MouseEvent("mousemove", {
+        bubbles: true,
+        cancelable: true,
+        clientX: x,
+        clientY: y,
+        buttons: 1,
+      });
+      el.dispatchEvent(moveEvt);
+    }
+    if ((window as any).__REMOTE__) {
+      (window as any).__REMOTE__.dragCamera(dx, dy);
```

Comment:
should be options inside assist function

`tracker.use(assist({ dragCamera: dragFn, draggableFlag: default = 'draggable' }))`

inside: `hasOpenreplayAttribute(draggableFlag)` -> 'data-openreplay-${draggableFlag}'